File: src/rngGame/stats/Rarity.java
package rngGame.stats;

public enum Rarity {

	COMMON,
	UNCOMMON,
	RARE,
	VERY_RARE,
	EPIC,
	LEGENDARY,
	GOD,
	VOID;


	public String getTextureName() {
		return toString().toCharArray()[0] + toString().substring(1).toLowerCase();
	}

}

File: src/rngGame/stats/Rarity.kt
package rngGame.stats

enum class Rarity {

	COMMON,
	UNCOMMON,
	RARE,
	VERY_RARE,
	EPIC,
	LEGENDARY,
	GOD,
	VOID;

	@JvmField
	val textureName: String = "${toString().first()}${toString().drop(1).lowercase()}"

}
